hedera-node/hedera-app-api/src/main/module-info.java

```diff
@@ -0,0 +1,3 @@
+module com.hedera.hashgraph.app.api {
```

Comment:
These files must be moved under `src/main/java` as opposed to the current location of `src/main` which places these files outside of all source sets.